### ISSUE TITLE
Turn off auto-scale? 

### ISSUE THREAD

**Issue by arw36** [ms609/Ternary#83]
Does the ternary plot auto scale the three variables? I'm dropping some other variables so I can focus on the top three. So for a few points, there will be a <100 sum across the three. But it seems to be auto pushing my 70,0,0 to 100,0,0. 

Thanks - this is a cool package and exactly what I was looking for. 

![image](https://github.com/ms609/Ternary/assets/16709317/75f12e89-87a4-4dce-b9f0-f36fad568299)

**Comment by ms609** [comment]
Yes, it auto-scales to a 100% sum; if there are other components, points are in effect projected onto a plane of the higher-dimensional hypertetrahedron (?) corresponding to all available dimensions.

There is more than one way to make this projection; what is your desired behaviour in the case you describe?

**Comment by arw36** [comment]
My desired behavior is on the triangle plot if given values (60, 20, 10) that it accurately positions at 60,20,10 even though these do not sum to 100. This effectively eliminates the 10 value that may go to another category not shown on the triangle plot. In my case, I am focused on diet items from arthropoda, chordata, and arthropoda and eliminating all other categories, but do not want to rescale these categories in relation to only each other.

**Comment by ms609** [comment]
60, 20, 10 does not correspond to a position on the triangle.  Perhaps you could show on an image where you would like this point to plot?

**Comment by arw36** [comment]
I guess a more extreme would be 60, 0, 0 which I see as the blue dot. I would suspect a 60,20,10 to look something like the red dot. 

<img width="606" alt="example" src="https://github.com/ms609/Ternary/assets/16709317/59bee0d6-b4ef-4c80-8587-14f06f5a32ae">

A separate goal, was to have the inset triangle which would show no single category comprises >60 value, but I also don't think that accurately visualizes this boundary.

**Comment by ms609** [comment]
60, 0, 0 is probably a clearer case to start with.  What is the basis for plotting 60, 0, 0 at 50, 50, 0 rather than (say) 50, 0, 50?

**Comment by arw36** [comment]
Oops, that was an error. It was meant to be, I guess, by that coordinate system 60,40,0. (I just put right at 60 on the axis, ignoring this intersection actually means with 40 input from another category? Can you show the alternative option?

**Comment by arw36** [comment]
I also suspect, since I'm violating the"sum to a constant" it may not be appropriate....

**Comment by ms609** [comment]
60, 0, 40 would correspond to the green circle:
![image](https://github.com/ms609/Ternary/assets/1695515/c313b979-4342-475b-bbe4-6f5b55df8c69)

**Comment by arw36** [comment]
Yes, I guess I can either clarify the incompleteness of the sample by using a pie chart or size-variation/color as the point marker or I could just use a full cartesian plot since my points aren't restricted to this plane. 

Thanks for your help.

**Comment by arw36** [comment]
Perhaps a warning message mentioning the re-scaling would be helpful? Is the default to rescale or drop values? And can you change this preference?
